File: BE/API/Models/DetallePresupuesto.cs
namespace API.Models;

public class DetallePresupuesto
{
    public Guid Id { get; set; }
    public int CantidadProductos { get; set; }
    // public double PrecioUnitario { get; set; }
    // public double Total { get; set; }
    public Guid PresupuestoId { get; set; }
    public Guid ProductoId { get; set; }
    public Producto? Producto { get; set; }
}

File: FE/src/interfaces/detallePresupuesto.ts
import { Guid } from "guid-typescript";
import { Producto } from "./producto";

export interface DetallePresupuesto{
    Id: string,
    CantidadProductos: number,
    PrecioUnitario: number,
    Total: number,
    ProductoId: string,
    PresupuestoId: string,
    Producto: Producto
}